### PR TITLE
Update to some minor details of the current VOPRF draft

### PR DESCRIPTION
**Pull request by claucece**
cc./ @chris-wood

### CODE REVIEW

**Review by chris-wood — CHANGES_REQUESTED** [review]
Thanks! I've been meaning to get around to this. I think some things regressed, which I've noted. Otherwise, this is good.

File: draft-irtf-cfrg-opaque.md [hunk 1]
```diff
@@ -361,7 +362,7 @@ OPAQUE relies on the following protocols and primitives:
 Note that we only need the base mode variant (as opposed to the verifiable mode
 variant) of the OPRF described in {{I-D.irtf-cfrg-voprf}}. We also assume the
 existence of a function `KeyGen` from {{I-D.irtf-cfrg-voprf}}, which
-generates an OPRF private and public key. OPAQUE only requires an OPRF private key.
+generates an OPRF private key. OPAQUE only requires an OPRF private key.
```
[the remark this answers is not in the document]
Comment (claucece) [reply draft-irtf-cfrg-opaque.md:365]: I'll revert it ;)

File: draft-irtf-cfrg-opaque.md [hunk 2]
```diff
@@ -521,8 +522,7 @@ struct {
 ~~~
 
 data
-: An encoded element in the OPRF group. See {{I-D.irtf-cfrg-voprf}} for a
-description of this encoding.
+: An encoded element in the OPRF group.
```
[the remark this answers is not in the document]
Comment (claucece) [reply draft-irtf-cfrg-opaque.md:525]: The data returned from the OPRF functions is always already encoded/serialized, so no need to specify how to encode.

File: draft-irtf-cfrg-opaque.md [hunk 3]
```diff
@@ -868,7 +859,7 @@ correspond to idU and idS. Thus, it is essential for the parties to agree on suc
 identities, including an agreed bit representation of these identities as needed.
 
 Applications may have different policies about how and when identities are
-determined. A natural approach is to tie idU to the identity the server uses
+determined. A natural approach is to tie idU to the identity the user uses
```
[the remark this answers is not in the document]
Comment (claucece) [reply draft-irtf-cfrg-opaque.md:862]: Ah, ok. Reverting this.

File: draft-irtf-cfrg-opaque.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (claucece) [reply draft-irtf-cfrg-opaque.md:525]: Except for the scalar that is returned from `blind`.

File: draft-irtf-cfrg-opaque.md [hunk 4]
```diff
@@ -581,10 +579,9 @@ Output:
 
 Steps:
 1. (r, M) = Blind(pwdU)
-2. data = Serialize(M)
-3. Create RegistrationRequest request with data
-4. Create RequestMetadata metadata with Serialize(r)
-5. Output (request, metadata)
+2. Create RegistrationRequest request with M
+3. Create RequestMetadata metadata with Serialize(r)
```
[the remark this answers is not in the document]
Comment (claucece) [reply draft-irtf-cfrg-opaque.md:583]: Agreed ;)

File: draft-irtf-cfrg-opaque.md [hunk 5]
```diff
@@ -735,6 +730,7 @@ of this integration.
 
 ~~~
 struct {
+    opaque id<1..2^16-1>; /* optional */
```
[the remark this answers is not in the document]
Comment (claucece) [reply draft-irtf-cfrg-opaque.md]: Done.

File: draft-irtf-cfrg-opaque.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (chris-wood) [reply draft-irtf-cfrg-opaque.md:525]: 👍

File: draft-irtf-cfrg-opaque.md [hunk 6]
```diff
@@ -735,6 +730,7 @@ of this integration.
 
 ~~~
 struct {
+    opaque id<1..2^16-1>;
```
Comment (chris-wood) [inline draft-irtf-cfrg-opaque.md:733]:
```suggestion
```

**Review by chris-wood — APPROVED** [review]

**Pull request merged**